### ISSUE TITLE
`s2b` bug in Node.js

### ISSUE THREAD

**Issue by dittos** [tomitribe/jwt-js-decode#39]
`s2b` returns wrong encoding in Node.js if char code is greater than 127.

```
$ node
Welcome to Node.js v16.15.1.
Type ".help" for more information.
> require('jwt-js-decode').s2b('\xcb\xf9')
'w4vDuQ=='  <- wrong
> btoa('\xcb\xf9')
'y/k='  <- OK
```

**Comment by dblevins** [comment]
@dittos Thank you!  I'm not one of the developers who normally works on this repo, but just wanted to say contributions are very welcome!  The more the merrier :)

**Comment by Dexmaster** [comment]
Fixed in version `1.7.0` https://www.npmjs.com/package/jwt-js-decode/v/1.7.0
